eq-author/src/App/questionPage/Design/QuestionPageEditor/AdditionalInfo.js

```diff
@@ -0,0 +1,80 @@
+import React from "react";
+import { withApollo } from "react-apollo";
+import PropTypes from "prop-types";
+import { propType } from "graphql-anywhere";
+import { flowRight } from "lodash";
+
+import WrappingInput from "components/Forms/WrappingInput";
+import RichTextEditor from "components/RichTextEditor";
+import { Field, Label } from "components/Forms";
+
+import withChangeUpdate from "enhancers/withChangeUpdate";
+
+import MultipleFieldEditor from "./MultipleFieldEditor";
+import withFetchAnswers from "./withFetchAnswers";
+
+import pageFragment from "graphql/fragments/page.graphql";
+
+const contentControls = {
+  bold: true,
+  emphasis: true,
+  piping: true,
+};
+
+export class StatelessAdditionalInfo extends React.Component {
+  render() {
+    const {
+      page,
+      onChange,
+      onUpdate,
+      fetchAnswers,
+      onChangeUpdate,
+    } = this.props;
+    return (
+      <MultipleFieldEditor label="Additional information">
+        <Field>
+          <Label htmlFor="additional-info-label">Label</Label>
+          <WrappingInput
+            id="additional-info-label"
```

Comment:
Do we still need this `id`? I thought we use `name`

Reply:
It's a required prop of `WrappingInput`.  It's not obvious why and I would be hesitant to make it optional. Do you want me to dig deeper?

Reply:
If its a 3rd party lib then don't bother otherwise yes please.

Reply:
The `id` is required for `label` elements as described here:

> The id of a labelable form-related element in the same document as the <label> element. The first element in the document with an id matching the value of the for attribute is the labeled control for this label element, if it is a labelable element.

Reply:
So its used on the label and input for accessibility. Makes sense.